### PR TITLE
Add tailwind black text color

### LINKED ISSUE #1469: Global audio player track title is incorrectly pink
## Problem

The global audio player is displaying the audio name link in pink text:

CleanShot 2022-05-24 at 06 55 32@2x

## Description

This text should be black, per the designs.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by VincentvI023**
## Fixes

Fixes #1469 by @zackkrida 

## Description

Adding a new tailwind class text-dark-charcoal to change the color of the global audio player

image

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ x I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by krysal — APPROVED** [review]
Looks good to me. Thanks for your contribution @VincentvI023!

Maybe it should be pink when is on hover? I will let @panchovm or @zackkrida tell.

**Comment by zackkrida** [comment]
I think an underline on hover would look best

File: src/components/VAudioTrack/layouts/VGlobalLayout.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <div class="relative flex-grow">
       <VLink
         :href="`/audio/${audio.id}`"
-        class="absolute inset-x-0 z-10 top-[10.5px] px-4 flex flex-row items-center justify-between line-clamp-2 pe-12 text-sr font-semibold"
+        class="absolute inset-x-0 z-10 top-[10.5px] px-4 flex flex-row items-center justify-between line-clamp-2 pe-12 text-sr font-semibold text-dark-charcoal"
```
Comment (obulat) [inline src/components/VAudioTrack/layouts/VGlobalLayout.vue:11]:
```suggestion
        class="absolute inset-x-0 z-10 top-[10.5px] px-4 flex flex-row items-center justify-between line-clamp-2 pe-12 text-sr font-semibold text-dark-charcoal hover-underline"
```

**Review by obulat — CHANGES_REQUESTED** [review]
Thank you for your contribution, @VincentvI023!
I've added a suggestion for underlining the link on hover :)

**Review by obulat — APPROVED** [review]
Thank you for your contribution, @VincentvI023 ! Welcome to Openverse :)

**Pull request merged**